ansible/roles/centos7/dashboards/templates/opensearch_dashboards.yml

```diff
@@ -0,0 +1,13 @@
+server.port: 5601
+server.host: "{{ hostvars[inventory_hostname]['ip'] }}"
+opensearch.hosts: ["{{ os_nodes_dashboards }}"]
+opensearch.ssl.verificationMode: none
+opensearch.username: "kibanaserver"
```

Comment:
Can we use a different name here , maybe `dashboardserver`?

Reply:
@TheAlgo  `kibanaserver` is the name used in default configuration file from Opensearch dashboards source file. Refer: https://opensearch.org/docs/dashboards/install/tls/ 

I suggest we should follow the default configuration from installation source to avoid confusions. May we it should be changed first in source files and official documentations then we can change it.

Reply:
@peterzhuamazon Just curious , why are we using `kibanaserver` till now?

Reply:
@TheAlgo @peterzhuamazon `kibanaserver` is used in security plugin where it should be changed and then configuration. I raised the below PRs to fox this. Once this merged and released, then we can update here.

https://github.com/opensearch-project/security/pull/1443
https://github.com/opensearch-project/opensearch-build/pull/458

Reply:
We are still in discussion with Security Team on this @saravanan30erd please hold on for some time before we make a decision on this.

Thanks.

Reply:
@peterzhuamazon  Sure, I think we can continue with `kibanaserver` here as of now.

Reply:
We will keep this for now @TheAlgo.
@saravanan30erd please create an issue to track this.
Thanks.